src/js/lib/icons/src/index.js

```diff
@@ -122,184 +14,94 @@ class Logo {
     init() {
         var scaleStop = 128;
         var scaleLog = Math.log2(scaleStop);
-        this.borderWidth = this.borderMax - Math.pow(Math.log2(Math.min(this.size, 128)) / scaleLog, 3) * scaleLog;
+        this.borderWidth = 6 // this.borderMax - Math.pow(Math.log2(Math.min(this.size, 128)) / scaleLog, 3) * scaleLog;
```

Comment:
Why the code is commented? When we could return to the old version? Leave an explanation.